kinds/accesspolicy/access_policy_kind.cue

```diff
@@ -0,0 +1,33 @@
+package kind
+
+name:        "AccessPolicy"
+maturity:    "merged"
+description: "Access rules for a scope+role.  NOTE there is a unique constraint on role+scope"
+
+lineage: seqs: [
+	{
+		schemas: [
+			//0.0
+			{
+				// UID for user/team/service...
+				// ??? should this be a #ResourceID  but kind=user|team|???
+				role: string
+
+				// The entity this policy applies to
+				scope: #ResourceID
+
+				rules: [...#AccessRule]
+
+				#ResourceID: {
+					kind: string
+					name: string
+				} @cuetsy(kind="interface")
+
+				#AccessRule: {
+					what:   string // dashboards, dashboards.permissions, alert.rules, ...
```

Comment:
I'm torn on what to call this, it's not exactly a kind but it's close.  We express the action "Set permissions on a folder" as "folders.permissions:write" rather than "folders:write-permissions".

Where that particular example gets really interesting is that permissions on a folder will be represented as an access policy that's scoped to the folder, so that permission is more like accesspolicy:write with a caveat that the accesspolicy must have a scope targeting the folder.